### PR TITLE
Update mail docs

### PR DESCRIPTION
**Pull request by m1cr0man**
Adds new documentation on the mail infrastructure.

### CODE REVIEW

**Review by benmcmahon100 — COMMENTED** [review]
I would like to see an overview of what each component does for the uninitiated as the root of this. I like everything else though

**Comment by m1cr0man** [comment]
The terminology explains what role each service fulfils.

**Comment by benmcmahon100** [comment]
Ah yes I see now it's not where I expected to see it

**Review by benmcmahon100 — DISMISSED** [review]

File: docs/index.md [hunk 1]
```diff
@@ -44,10 +44,7 @@ Day to day running of things.
 
 #### Mail
 
-- [Exim email setup](services/exim)
-- [Dovecot IMAP](services/dovecot)
-- [Mailing Lists (Mailman)](services/mailman)
-- [Mail setup for mutt](procedures/mail_setup)
+- [Mail docs index](mail)
```
Comment (butlerx) [inline docs/index.md]: can we move `mail` and `web` under services?

File: docs/procedures/newboard.md [hunk 2]
```diff
@@ -1,10 +0,0 @@
-# New board
-
-Sup dawg, so I herd you like boards?
-
-So `ctlinnd newgroup board.name y` will put a new board in your boards so you
```
Comment (butlerx) [inline docs/procedures/newboard.md]: This doc is still correct unless boards have changes?

Comment (m1cr0man) [reply docs/procedures/newboard.md]: Whoops. Will restore

**Review by butlerx — DISMISSED** [review]

**Comment by m1cr0man** [comment]
@butlerx could you approve again and squash merge please?

**Review by butlerx — APPROVED** [review]

**Pull request merged**